utils/src/integ-test/java/uk/gov/moj/sdt/test/utils/IExecuteScriptService.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.moj.sdt.test.utils;
+
+import org.springframework.core.io.Resource;
+
+import java.util.concurrent.Future;
+
+public interface IExecuteScriptService {
+
+    public boolean executeScript(Resource script);
+
+    public Future<Boolean> runScript(Resource script);
```

Comment:
JavaDoc comments would be useful here to explain the difference between the two methods.  This would help someone understand why they would use one method over the other.